### PR TITLE
Support JupyterLab 4.0

### LINKED ISSUE #41: Update to JupyterLab 4
## Is your feature request related to a problem? Please describe.

The extension is built for JupyterLab 3 for now.

## Describe the solution you'd like

JupyterLab 4 support.

Probably not much to be done. Maybe updating to the latest extension template would be useful to grab all the latest changes: https://github.com/jupyterlab/extension-template

## Additional context

This would be useful for JupyterLite 0.2.0: https://github.com/jupyterlite/jupyterlite/releases/tag/v0.2.0a0

### PR DESCRIPTION
**Pull request by mengyanw**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,88 +1,199 @@
...
+        "outputDir": "jupyterlab_cell_flash/labextension"
+    },
+    "jupyter-releaser": {
+        "hooks": {
+            "before-build-npm": [
+                "python -m pip install jupyterlab~=3.1",
+                "jlpm"
+            ]
+        }
+    },
```
Comment (jtpio) [inline package.json]: We can probably remove that section from `package.json` as the releaser hooks are already defined in `pyproject.toml`.

```suggestion
```

**Comment by jtpio** [comment]
Thanks @mengyanw for looking into this!

Looks like this needs a quick lint pass, which should help fix CI. Hopefully running `jlpm lint` should do it.

**Comment by mengyanw** [comment]
@jtpio Thank you! Updated.

**Review by jtpio — APPROVED** [review]
Thanks!

**Comment by jtpio** [comment]
Thanks again @mengyanw!

This should now be available in the new version: https://github.com/jupyterlab-contrib/jupyterlab-cell-flash/releases/tag/v0.4.0

**Pull request merged**